src/app/query-home/search-area/editor/query-editor.tsx

```diff
@@ -34,7 +34,14 @@ const QueryEditor = ({value, disabled}: Props) => {
   }
   const ref = useEditorView({value, disabled, onChange})
 
-  return <EditorWrap isDisabled={disabled} ref={ref} onKeyDown={onKeyDown} />
+  return (
+    <EditorWrap
+      role="editor"
```

Comment:
Looks like most of the roles specified should actually be aria-labels:

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label.